hetzner/README-cloud.md

```diff
@@ -1,17 +1,82 @@
 # Hetzner Cloud
 
 The following guide details how to provision a Hetzner VM using the [Hetzner Cloud APi](https://docs.hetzner.cloud/#overview) or the  [Hetzner Cloud client](https://github.com/hetznercloud/cli) that you can install using the 
-following brew command `brew install hcloud`
+following brew command `brew install hcloud`.
+
+For Linux installation, binaries are available at [releases](https://github.com/hetznercloud/cli/releases). Download `hcloud-linux-amd64.tar.gz` and extract the `hcloud` 
+file into the `~/bin` directory. 
+
+All Ansible playbooks must be launched from the root of the project, at `k8s-infra`.
+
+## Init the hetzner context
+
 To getting started, you must get a Token for your API as described [here](https://docs.hetzner.cloud/#overview-getting-started).
 
 In order to create a vm and next access it, you must first import your ssh public key using this command
 ```bash
 hcloud ssh-key create --name USER_KEY_NAME --public-key-from-file ~/.ssh/id_rsa.pub
```

Comment:
Is it still needed that we use this command as we use a shared ssh key when the vm is created ? If no, then disregard this section please

Reply:
The playbooks still use the local ssh key, the integration with pass is not implemented yet.

Reply:
We have created a shared ssh key for the team -> snowdrop. So it is not needed to import the ssh key of the user. See our spring boot team info google doc -> Cloud Hetzner